### DISCUSSION TITLE
Could not resolve host: apktool.org

### DISCUSSION THREAD

**Discussion by 4ft35t** [iBotPeaches/Apktool#3276] [General]
```zsh
╭─root@ubuntu ~
╰─# nslookup apktool.org 1.1.1.1
Server:		1.1.1.1
Address:	1.1.1.1#53

** server can't find apktool.org: SERVFAIL

╭─root@ubuntu ~
╰─# nslookup apktool.org 8.8.8.8                                                                                           1 ↵
Server:		8.8.8.8
Address:	8.8.8.8#53

** server can't find apktool.org: SERVFAIL
```

**Comment by 4ft35t** [comment]
use https://dnschecker.org/#A/apktool.org to check

**Comment by iBotPeaches** [comment]
Wonderful. Not sure why this happened. Domain still under my control.

**Reply by iBotPeaches** [reply]
Well this sucks. I'm on vacation and don't really have the keys/2fa things in easy reach to tweak my DNS settings. It might be offline for the weekend - sorry for inconvenience.

Guess my move from Google -> Porkbun blew out the ALIAS/CNAME I had, presumably because I was doing the DNS via Google and Porkbun just moved those NS, which were terminated when the transfer completed. My mistake.

**Comment by iBotPeaches** [comment]
Found some time - setup the A, AAAA and CNAME on www. Unfortunately I'll have to wait for dns propagation to move from Google to Linode.

**Comment by iBotPeaches** [comment]
Closing as resolved. Many mistakes were made.

I transfered Google to Porkbun, unaware my NS records for this domain were on Google.

Google killed serving those NS records and they all disappeared, which led to this issue.

So I used Porkbun and setup the records using the new ALIAS record. Nothing was resolving after a full day of vacation so I figured that new type wasn't fully supported.

I changed to the Github A records, but I couldn't use more than 1 A record. So decided to move NS records again, this time to Linode where I host other DNS.

Once I checked the next day, I saw partial success, but mostly failure. I looked and saw it was DNSSEC validation failure. I had failed to disable DNSSEC prior to transfer so I thought I was screwed.

Turns out I could just remove the DNSSEC record from Porkbun as Linode didn't support natively generating them. All was resolved shortly after.

Lesson learned. Don't do migrations prior to vacation.
